cli/src/commands/deploy.ts

```diff
@@ -73,6 +73,12 @@ export default class Deploy extends Command {
         shell: process.platform === 'win32',
       });
 
+      await copy(pathResolve(projectDir, 'backend'), pathResolve(stagingDir, 'backend'), {
```

Comment:
Frustrating that this has to essentially be duplicated. Not really solvable short term unfortunately.

Reply:
There are subtle differences in the code around it so deduplication won't help that much here...

Reply:
That's why I said frustrating. At the end of the day, the code is very similar, minus subtle differences. Thats where the "Frustrating" came from.